broker/tasks/cloudwatch.py

```diff
@@ -137,6 +136,10 @@ def _create_health_check_alarm(health_check_id, tags) -> str:
     alarm_name = _get_alarm_name(health_check_id)
 
     # create alarm
+    kwargs = {}
+    if tags is not None:
```

Comment:
is it possible we'll get into a state where `service_instance.tags` == `[]`?
if so, we should change this to `if tags:`